distribution/lib/Standard/Base/0.0.0-dev/src/Data/Text/Extensions.enso

```diff
@@ -1268,6 +1334,72 @@ Text.locate_all self term="" matcher=Text_Matcher.Case_Sensitive = if term.is_em
             Nothing -> []
             matches -> matches.map m-> m.span 0 . to_grapheme_span
 
+## Find the first index of the `term` in the input.
+   Returns `Nothing` if the term was not found in the input.
+
+   Use `locate` if you need to know the length of the match as well.
```

Comment:
Ok, can we at least mention that the match length may be not equal to term length?

My point is - without any mention the user won't even know they may need to use `locate` because they may think "why use it, I can just do `term.length`".

```suggestion
## Find the first index of the `term` in the input.
   Returns `Nothing` if the term was not found in the input.

   Using `locate` is preferred as then you can also get the length of the match which may not necessarily be equal to the length of the searched term - not only in case of regex matching but also in case insensitive mode. See the Match Length section of `locate` for an explanation.
```